config/site/src/_includes/filtering_predicate_definitions/conjunctions.md

```diff
@@ -6,11 +6,11 @@
   filters that can't be provided on a single filter input because of collisions between key names.
   For example, if you want to provide multiple `anySatisfy: ...` filters, you could do `allOf: [{anySatisfy: ...}, {anySatisfy: ...}]`.
 
-  Will be ignored when `null` or an empty list is passed.
+  Will be treated as `true` when `null`, an empty object or list is passed.
```

Comment:
Do we want to mention the empty object case here? The schema type system doesn't allow empty object here, but it gets coerced from `{}` into `[{}]`. I don't really want to encourage users to not pass an object here...

Reply:
Makes sense, I have removed the mention of empty object.